airbyte-ci/connectors/auto_merge/src/auto_merge/pr_validators.py

```diff
@@ -48,12 +48,13 @@ def head_commit_passes_all_required_checks(
     return True, None
 
 
-# A PR is considered auto-mergeable if:
+# A standard PR is considered auto-mergeable if:
 #     - it has the AUTO_MERGE_LABEL
 #     - it targets the BASE_BRANCH
 #     - it touches only files in CONNECTOR_PATH_PREFIXES
 #     - the head commit passes all required checks
 
 # PLEASE BE CAREFUL OF THE VALIDATOR ORDERING
-# Let's declared faster checks first as the  check_if_pr_is_auto_mergeable function fails fast.
-ENABLED_VALIDATORS = [has_auto_merge_label, targets_main_branch, only_modifies_connectors, head_commit_passes_all_required_checks]
+# Let's declare faster checks first as the check_if_pr_is_auto_mergeable function fails fast.
+DEFAULT_ENABLED_VALIDATORS = [has_auto_merge_label, targets_main_branch, only_modifies_connectors, head_commit_passes_all_required_checks]
+PROMOTED_RC_PR_VALIDATORS = [has_auto_merge_label, targets_main_branch, only_modifies_connectors]
```

Comment:
nit: I'd reverse the logic.
The default validators would have less requirements than a second one which would require the head commit to pass.
```
DEFAULT_ENABLED_VALIDATORS = {has_auto_merge_label, targets_main_branch, only_modifies_connectors}
PASSING_CHECKS_VALIDATORS = DEFAULT_ENABLED_VALIDATORS + { "head_commit_passes_all_required_checks"}
```

It would define an implicit logic of strictness elevation.

Reply:
I agree -- updated.